components/ui/Sidebar.tsx

```
@@ -0,0 +1,133 @@
+
+      {/* Collapsed view - just icons */}
+      {collapsed && (
+        <div className="p-2 flex flex-col items-center gap-2">
+          {documents.slice(0, 5).map((doc) => (
+            <button
+              key={doc.id}
+              onClick={() => onDocumentClick(doc.id)}
+              className="w-10 h-10 flex items-center justify-center rounded-md text-[#E8DCC4] hover:bg-[#3D3426] transition-colors"
+              title={doc.title}
+            >
+              📄
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)